testing/environments/prev-minor.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:8.0.0-rc1
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.1.0-rc1
```

Comment:
@klacabane @matschaffer This looks off. Likely it should be 8.0.1 or similar? Same below.

Reply:
It seems the script is fine it is just not able to deal with -rc1 etc. As having rc1 etc. is kind of unique and an edge case, I suggest we clean it up manually.

Reply:
@matschaffer For the ping, I assumed this is because of stack monitoring that we test prev minor but it seems more generic, so we need to handle this from the Agent team.